resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -128,13 +130,46 @@ if [[ "$os" = 'el' || "$os" = 'sles' || "$os" = 'fedora' ]]; then
                 params+=("--additional-dependency")
                 params+=("<%= dep -%>")
         <% end -%>
+        # get rpm install trigger scripts
+        <% EZBake::Config[:redhat][:postinst_install_triggers].each do |package, script|
+             rpm_paths.each do |path| -%>
+              cat > <%=path %>/<%=package%>_postinst_triggers <<EOM
+              if [ "\$1" -eq 1 ] ; then
+                 "<%= script.join("\n") %>"
+                 :
+              fi
+EOM
+            <% end -%>
+            params+=("--rpm-trigger" "'<%=package%>: $dir/<%=package%>_postinst_triggers'")
+        <% end -%>
+        # get rpm upgrade trigger scripts
+        <% EZBake::Config[:redhat][:postinst_upgrade_triggers].each do |package, script|
+             rpm_paths.each do |path| -%>
+             cat > <%=path %>/<%=package%>_postinst_triggers <<EOM
+              if [ "\$1" -gt 1 ] ; then
+                 "<%= script.join("\n") %>"
+                 :
+              fi
+EOM
+               <% end -%>
+            params+=("--rpm-trigger" "'<%=package%>: $dir/<%=package%>_postinst_triggers'")
+        <% end -%>
         : # Need something in case there are no additional dependencies
 else
         # if we aren't an rpm, pull in deb dependencies
         <%EZBake::Config[:debian][:additional_dependencies].each do |dep| -%>
                 params+=("--additional-dependency")
                 params+=("<%= dep -%>")
         <% end -%>
+        <%EZBake::Config[:debian][:interested_install_triggers].each do |interest_name, _| -%>
+                params+=("--deb-interest-trigger" "<%=interest_name-%>")
+        <% end -%>
+        <%EZBake::Config[:debian][:interested_upgrade_triggers].each do |interest_name, _| -%>
+                params+=("--deb-interest-trigger" "<%=interest_name-%>")
+        <% end -%>
+        <%EZBake::Config[:debian][:activated_triggers].each do |activated_name| -%>
+                params+=("--deb-activate-trigger" "<%=activated_name-%>")
+        <% end -%>
```

Comment:
you got some deb stuff in your rpm PR :)

Reply:
(in other places too)

Reply:
it looks like there's just an extra commit in this branch